Week2/homework/map_filter.js

```diff
@@ -1,5 +1,13 @@
-'use strict';
-
 const numbers = [1, 2, 3, 4];
+// const newNuxmbers = [];
+
+// for (let i = 0; i < numbers.length; i++) {
+//   if (numbers[i] % 2 !== 0) {
+//     newNumbers.push(numbers[i] * 2);
+//   }
+// }
 
-// Add your code here
+let newNumbers = numbers
+    .filter(number => number % 2 !== 0) 
+    .map(number => number * 2);
+console.log('The doubled numbers are', newNumbers);
```

Comment:
Good job.  As an extra challenge.  How would you implement this with Array.reduce? 😄

Reply:
Hi Rohni,
Thank for spending time to review my homework!

Reply:
I honestly have a memory issue...
I could not remember what was the way of thinking I followed last week...
Maybe I need to refresh my my thoughts with some tutorial videos and will replay to you asap!
Thanks again!!!